node/libs/roles/src/validator/testonly.rs

```diff
@@ -22,8 +23,13 @@ impl Setup {
     /// New `Setup` with a given `fork`.
     pub fn new_with_fork(rng: &mut impl Rng, validators: usize, fork: Fork) -> Self {
         let keys: Vec<SecretKey> = (0..validators).map(|_| rng.gen()).collect();
+        let weight = 1;
         let genesis = Genesis {
-            validators: ValidatorSet::new(keys.iter().map(|k| k.public())).unwrap(),
+            validators: ValidatorCommittee::new(keys.iter().map(|k| WeightedValidator {
```

Comment:
nit: we can shorten "ValidatorCommittee" to just "Committee".

Reply:
I like that, but this was a suggestion from @brunoffranca. Do you agree, Bruno?

Reply:
to clarify: we will use it as "validator::Commitee" all over the place anyway.